Vertexvis/vertex-web-sdk
# [PLAT-2432] Add walk mode support

## Summary

Adds support for walk mode by exposing a couple new tools, specifically the `<vertex-viewer-walk-mode-tool>` and `<vertex-viewer-teleport-tool>` with the thought that these can be dropped in underneath a `<vertex-viewer>` and toggled on/off.

## Test Plan

- Verify the `<vertex-viewer-walk-mode-tool>`'s behavior
  - Should be able to walk through the model with `enabled=true` (default)
  - Should be able to provide a custom `<vertex-viewer-teleport-tool>` to set animation configurations
  - Should be able to enable/disable
  - Should be able to change the `teleportTool` and have it update the underlying `<vertex-viewer-teleport-tool>`
- Verify the `<vertex-viewer-teleport-tool>`'s behavior
  - Should be able to change the `tool` and see the behavior change
  - Should be able to provide custom animation configuration

## Release Notes

N/A

## Possible Regressions

Interaction APIs - small change to update `transformCamera` in the perspective/orthographic APIs + added support for a render configuration

## Dependencies

N/A
